<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>405</width>
                <height>451</height>
            </rect>
        </property>
        <property name="font">
            <font>
                <pointsize>14</pointsize>
            </font>
        </property>
        <property name="windowTitle">
            <string>Gestionar etiquetas</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QGridLayout" name="gridLayout">
                <item row="2" column="0">
                    <widget class="QLabel" name="label_etiquetas">
                        <property name="text">
                            <string>Etiquetas</string>
                        </property>
                    </widget>
                </item>
                <item row="3" column="0" colspan="2">
                    <widget class="QTableWidget" name="tableWidget">
                        <column>
                            <property name="text">
                                <string>Etiqueta</string>
                            </property>
                        </column>
                    </widget>
                </item>
                <item row="4" column="0">
                    <widget class="QPushButton" name="pushButton_eliminar_etiqueta">
                        <property name="text">
                            <string>Eliminar etiqueta</string>
                        </property>
                    </widget>
                </item>
                <item row="0" column="0">
                    <widget class="QLabel" name="label_addlabel">
                        <property name="text">
                            <string>AÃ±adir etiqueta</string>
                        </property>
                    </widget>
                </item>
                <item row="4" column="2">
                    <widget class="QPushButton" name="pushButton_cerrar">
                        <property name="text">
                            <string>Cerrar</string>
                        </property>
                    </widget>
                </item>
                <item row="4" column="1">
                    <widget class="QPushButton" name="pushButton_guardar_cambios">
                        <property name="text">
                            <string>Guardar cambios</string>
                        </property>
                    </widget>
                </item>
                <item row="1" column="0">
                    <widget class="QLineEdit" name="lineEdit_nuevaetiqueta"/>
                </item>
                <item row="1" column="1">
                    <widget class="QPushButton" name="pushButton_add">
                        <property name="text">
                            <string>+</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>405</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
